src/views/VideoMania/index.jsx

```
@@ -64,24 +81,53 @@ const VideoMania = () => {
+          ))
+        )}
+      </div>
+
+      {/* Top Video*/}
+      <div className="flex items-center mt-10">
+        {isTopVideoFetching ? (
+          <Spinner />
+        ) : topVideo ? (
+          <div className="flex justify-center items-center gap-5">
+            <div className="border border-gray-200 p-2 rounded-md">
+              <img src={videoIcon} className="w-32 h-28 overflow-hidden" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)